MobileWallet/Common/Extensions/Animation+EnumInit.swift

```diff
@@ -54,6 +54,9 @@ extension Animation {
         case nerdEmoji = "NerdEmojiAnimation"
 
         case waveEmoji = "WaveEmojiAnimation"
+
+        case checkboxSelectAnimation = "checkbox-selectAnimation"
```

Comment:
Follow the naming convention in this file so far: `CheckboxSelectAnimation`, `CheckboxDeselectAnimation`.